Vagrantfile

```diff
@@ -15,7 +15,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # options are documented and commented below. For a complete reference,
   # please see the online documentation at vagrantup.com.
 
-  config.vm.box = "generic/ubuntu2004"
+  # Not available in the generic repository yet.
+  #config.vm.box = "generic/ubuntu2404"
+  config.vm.box = "alvistack/ubuntu-24.04"
```

Comment:
Hrm, seems like there might not be any official boxes anymore: https://askubuntu.com/a/1521304/164829

Another reason to use LXC I suppose! (for Linux users).

Reply:
> Another reason to use LXC I suppose! (for Linux users).

So what will you do on Mac?

I tend to ignore Windows because they can run Linux in any virtual environment and go from there. But Mac is a unix and popular amongst a lot of devs. I wouldn't mind you switching to Linux though. :wink:

Reply:
I did actually consider staying on Mac so that we can maintain support for all environments that contributors may have (yes, even the proprietary closed Apple ecosystem). I

I started looking yesterday and came across [Macpine](https://beringresearch.github.io/macpine/) which might be a good option, using Macpine